go/types/map.go

```diff
@@ -227,6 +227,18 @@ func (m Map) IterAll(cb mapIterAllCallback) {
 	})
 }
 
+func (m Map) IterRange(start, end Value, cb mapIterAllCallback) {
```

Comment:
I did see that comment. I had remembered it as just getting rid of the includeStartValue and includeEndValue flags.  But yeah, I think IterFrom() is better. I'll make the change.